pages/feed/index.tsx

```diff
@@ -78,6 +79,8 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
 
   const { data: followersRepo } = useFetchFollowersHighlightRepos();
 
+  const isDesktop = useMediaQuery("(min-width: 1280px)");
```

Comment:
Using this to prevent the top user panel component from rendering on mobile

This also prevents it from checking for following on mobile and tab